cmd/traefik/plugins.go

```diff
@@ -35,12 +35,12 @@ func initPlugins(staticCfg *static.Configuration) (*plugins.Client, map[string]p
 		var err error
 		client, err = plugins.NewClient(opts)
 		if err != nil {
-			return nil, nil, nil, err
+			return nil, nil, nil, fmt.Errorf("failed to create plugins client: %w", err)
```

Comment:
I don't really understand those suggestions because all the other error message use `failed to` and not `unable to`

Reply:
We use both forms in the codebase, and I tend to prefer the `unable to` form, if you don't want to change it I can live with that 😄 

To be honest I would prefer to remove all the `unable to` in "sub-errors" and keep only the upper one to avoid having things like: `unable to ...: unable to ...`, to some like `unable to ...: action: ...`

Reply:
I updated all the messages of the plugin section to use `unable to`.

Reply:
Thanks